.github/workflows/GHA-Scala-Functional-Tests.yaml

```diff
@@ -95,7 +95,7 @@ jobs:
           TEST_TYPE: functional
         run: |
           # Removed ":newrelic-cats-effect3-api:test" temporarily
-          ./gradlew --console=plain --parallel :newrelic-scala-api:test :newrelic-scala-cats-api:test :newrelic-scala-zio-api:test -Ptest${{ matrix.java-version }} --continue --scan -Dscan.uploadInBackground=false --build-cache
```

Comment:
Chatted to Gerard about this, currently failing as the new submodule newrelic-scala-monix-api can't be found although tests are passing when run locally

Reply:
GHA issues mentioned to @twcrone